code/modules/unit_tests/reagent_mod_expose.dm

```diff
@@ -0,0 +1,50 @@
+// testing the mob expose procs are working
+
+/datum/reagent/method_patch_test
+	name = "method patch test"
+
+/datum/reagent/method_patch_test/expose_mob(mob/living/target, methods=PATCH, reac_volume, show_message = TRUE)
+	. = ..()
+	if(methods & PATCH)
+		target.health = 90
+	if(methods & INJECT)
+		target.health = 80
+
+/datum/unit_test/reagent_mob_expose/Run()
+	var/mob/living/carbon/human/human = allocate(/mob/living/carbon/human)
+	var/obj/item/reagent_containers/dropper/dropper = allocate(/obj/item/reagent_containers/dropper)
+	var/obj/item/reagent_containers/food/drinks/drink = allocate(/obj/item/reagent_containers/food/drinks/bottle)
+	var/obj/item/reagent_containers/pill/patch/patch = allocate(/obj/item/reagent_containers/pill/patch)
+	var/obj/item/reagent_containers/syringe/syringe = allocate(/obj/item/reagent_containers/syringe)
+
+	// INGEST
+	TEST_ASSERT_EQUAL(human.fire_stacks, 0, "Human has fire stacks on allocate")
```

Comment:
```suggestion
	TEST_ASSERT_EQUAL(human.fire_stacks, 0, "Human has fire stacks before taking phlogiston")
```

Reply:
Cant complain with wording suggestions it always helps me